docs/tools/purge-cache-for-changed-files.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
it's more like antipattern to name python scripts with dashes because it's not a valid identifier, so it can't be imported without hacks